LDAR_Sim/model_code/default_parameters/default_global_parameters.py

```diff
@@ -25,12 +25,16 @@
     'input_directory': './inputs_template',
     'output_directory': './outputs',
     'n_processes':  None,
-    'n_simulations': 2,
     'print_from_simulations': True,
+    'n_simulations': 2,
     'spin_up': 0,
```

Comment:
Is spin_up removed here or in a different PR?

Reply:
included in this now